### DISCUSSION TITLE
Support for https://huggingface.co/CohereForAI/c4ai-command-r-plus

### DISCUSSION THREAD

**Discussion by bks5881** [vllm-project/vllm#3939] [Ideas]
Can you please add support or point me to a branch for https://huggingface.co/CohereForAI/c4ai-command-r-plus ? currently vllm fails to load and throws error. I updated transformers to the latest and still fails. i am able to load it directly with transformers though.

The error vllm throws is
KeyError: 'model.layers.14.self_attn.k_norm.weight' [repeated 2x across cluster]

**Comment by martin416** [comment]
how did you  load it in huggingface's transformers ,thanks

**Reply by bks5881** [reply]
@martin416  i used the latest transformers and installed from git, and used their code on hf.
